test/reporters/xunit.spec.js

```diff
@@ -320,6 +321,57 @@ describe('XUnit reporter', function() {
         expect(expectedWrite, 'to be', expectedTag);
       });
     });
+    describe('should write expected summary statistics', function() {
```

Comment:
LGTM. Although this needs to be `it(`

Reply:
Fixed the `describe` vs. `it` - thanks!

Although these changes test for that issue, the unit test should be there to ensure that all xunit outlook looks correct.